src/components/ActivityTimeline/__test__/activityTimeline.spec.js

```diff
@@ -12,4 +14,104 @@ describe('<ActivityTimeline/>', () => {
         );
         expect(component.find(Card).exists()).toBe(true);
     });
+    it('should fire onToggleSection with the right params when variant is accordion and multiple is false', () => {
+        const onToggleSectionFN = jest.fn();
+        const component = mount(
+            <ActivityTimeline variant="accordion" onToggleSection={onToggleSectionFN}>
+                <TimelineMarker name="marker1" />
+                <TimelineMarker name="marker2" />
+                <TimelineMarker name="marker3" />
+            </ActivityTimeline>,
+        );
+        component
+            .find(ButtonIcon)
+            .at(0)
+            .simulate('click');
+        expect(onToggleSectionFN).toHaveBeenCalledWith({
+            activeSectionNames: 'marker1',
+            toggledSection: 'marker1',
+        });
+        component
+            .find(ButtonIcon)
+            .at(2)
+            .simulate('click');
+        expect(onToggleSectionFN).toHaveBeenCalledWith({
+            activeSectionNames: 'marker3',
+            toggledSection: 'marker3',
+        });
+    });
+    it('should fire onToggleSection with the right params when variant is accordion and multiple is passed', () => {
+        const onToggleSectionFN = jest.fn();
+        const component = mount(
+            <ActivityTimeline variant="accordion" multiple onToggleSection={onToggleSectionFN}>
+                <TimelineMarker name="marker1" />
+                <TimelineMarker name="marker2" />
+                <TimelineMarker name="marker3" />
+            </ActivityTimeline>,
+        );
+        component
+            .find(ButtonIcon)
+            .at(0)
+            .simulate('click');
+        expect(onToggleSectionFN).toHaveBeenCalledWith({
+            activeSectionNames: ['marker1'],
+            toggledSection: 'marker1',
+        });
+        component
+            .find(ButtonIcon)
+            .at(2)
+            .simulate('click');
+        expect(onToggleSectionFN).toHaveBeenCalledWith({
+            activeSectionNames: ['marker1', 'marker3'],
+            toggledSection: 'marker3',
+        });
+        component
+            .find(ButtonIcon)
+            .at(2)
+            .simulate('click');
+        expect(onToggleSectionFN).toHaveBeenCalledWith({
+            activeSectionNames: ['marker1'],
+            toggledSection: 'marker3',
+        });
+    });
+    it('should expand the right marker when variant is accordion and its name is passed in activeSectionNames prop', () => {
```

Comment:
here we could use two describes to separate accordion and default variants, also avoiding the repetitive `when variant is accordion ...` in each test name